apps/api/src/app/events/usecases/trigger-event/message-filter.matcher.ts

```diff
@@ -157,8 +182,20 @@ async function getWebhookResponse(
   } catch (err) {
     // eslint-disable-next-line no-console
     if (err.response && err.response.body) {
-      // eslint-disable-next-line no-console
-      console.log('exception while performing webhook request - ', err.response.body);
+      await configuration.createExecutionDetails.execute(
+        CreateExecutionDetailsCommand.create({
+          ...CreateExecutionDetailsCommand.getDetailsFromJob(configuration.job),
+          detail: DetailEnum.WEBHOOK_FILTER_FAILED,
+          source: ExecutionDetailsSourceEnum.WEBHOOK,
+          status: ExecutionDetailsStatusEnum.PENDING,
```

Comment:
same here, I am not sure what source and status to add here, for me FAILED is a good idea but then an exception will be thrown, which I think is not a good idea here because then the job status won't be updated.